src/ncp/ncp_base.cpp

```diff
@@ -124,6 +124,7 @@ const NcpBase::GetPropertyHandlerEntry NcpBase::mGetPropertyHandlerTable[] =
     { SPINEL_PROP_PHY_CHAN, &NcpBase::GetPropertyHandler_PHY_CHAN },
     { SPINEL_PROP_PHY_RSSI, &NcpBase::GetPropertyHandler_PHY_RSSI },
     { SPINEL_PROP_PHY_TX_POWER, &NcpBase::GetPropertyHandler_PHY_TX_POWER },
+    { SPINEL_PROP_PHY_SENSITIVITY, &NcpBase::GetPropertyHandler_PHY_SENSITIVITY },
```

Comment:
You need to wrap all this new NPC code with `#if OPENTHREAD_ENABLE_RAW_LINK_API`.